[23-Mar-2019 12:14:54 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:14:58 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:14:59 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:15:37 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:15:40 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:15:40 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:16:08 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:18:54 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:18:58 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:19:12 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:19:20 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:19:39 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:22:43 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:22:50 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:23:44 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:27:48 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:27:51 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:27:59 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:28:35 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:38:13 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:40:49 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:41:01 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:41:02 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:41:03 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:41:05 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:41:08 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:42:23 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:42:40 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:42:57 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:43:57 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:43:58 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:44:11 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:44:12 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:44:14 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:44:20 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:44:21 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:44:21 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[23-Mar-2019 12:46:26 UTC] PHP Parse error:  syntax error, unexpected '?' in /home/lumbiniorg/test.lumbini.org.uk/charity/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php on line 500
[06-May-2019 14:40:43 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:40:46 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:40:48 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:41:20 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:41:23 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:41:25 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:41:26 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:41:31 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:42:12 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:43:49 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:43:59 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:44:02 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:44:21 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:44:47 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:46:25 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:46:34 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:47:01 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:47:55 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:49:24 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:49:49 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:49:52 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:50:58 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:51:01 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:51:06 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:51:12 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:51:50 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:54:19 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:55:04 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:55:55 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:55:57 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:56:00 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:56:58 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:57:00 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:57:01 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:57:02 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:57:03 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:57:04 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:58:21 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 14:58:30 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:00:42 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:02:17 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:02:19 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:02:21 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:02:24 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:02:29 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:02:31 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:02:32 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:02:38 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:02:39 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:03:24 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:03:27 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:03:30 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:03:32 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:04:01 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:04:30 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:04:38 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:04:42 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:04:43 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:04:43 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:07:38 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:07:40 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:07:56 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:07:58 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:08:00 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:08:31 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:08:33 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:09:35 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:09:36 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:09:36 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:09:37 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:09:37 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:09:38 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:09:38 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:09:38 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:09:39 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
[06-May-2019 15:09:40 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:234
Stack trace:
#0 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(397): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(374): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#2 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(289): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#3 /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(188): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
 in /home/lumbiniorg/charity/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 234
